Sources/MapboxCoreNavigation/EHorizon/RoadObject.swift

```diff
@@ -21,7 +21,7 @@ public struct RoadObject {
     public let location: RoadObjectLocation
 
     /** Type of the road object with metadata. */
-    public let type: RoadObjectType
+    public let type: RoadObject.ObjectType
```

Comment:
If we end up renaming the enumeration to `Kind`, then this property should also be renamed to `kind`.